Katello/katello
# Fixes #37632 - Use getDocsURL instead of useKatelloDocUrl

#### What are the changes introduced in this pull request?

In d400e3996f068841259b6d4274288f547e7a5f06 a useKatelloDocUrl helper was introduced. This should have never been introduced in the first place and instead should have been implemented in Foreman itself. Since Foreman 3.9.1 there is such an API so this should be used.

#### Considerations taken when implementing this change?

Downstream concern: https://github.com/RedHatSatellite/foreman_theme_satellite should also point to the correct URL.

#### What are the testing steps for this pull request?

* Open page
* Click documentation link
* Verify you end up on the correct documentation
